src/types/capsule.rs

```diff
@@ -151,6 +153,16 @@ impl PyCapsule {
 /// These methods are defined for the `Bound<'py, PyCapsule>` smart pointer, so to use method call
 /// syntax these methods are separated into a trait, because stable Rust does not yet support
 /// `arbitrary_self_types`.
+///
+/// # Name checking
+///
+/// Capsule methods contain arbitrary data which is cast to a specific type at runtime. This is
```

Comment:
Instead of "Capsule methods contain arbitrary data", this should maybe say "Capsules cointain arbitrary data" or, even more precisely, "Capsules contain pointers to arbitrary data"?